### PR TITLE
test for addroom page

### LINKED ISSUE #38: create test for addroom page

### PR DESCRIPTION
**Pull request by lyuboxin**
Resolved #38 create test for addroom page

### CODE REVIEW

File: test_addroom.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    date_start = browser.find_by_name("date_start")
+
+
+    browser.find_by_value('create').first.click()
+    checkresult(result)
+
+def checkresult(x):
+    resultMsg(browser.is_text_present(x))
+
+__testUrl = 'http://ubuntu1604-006.student.cs.uwaterloo.ca:19488/addroom.html'
```
Comment (zhangjiangqige) [inline test_addroom.py]: 19487?

Comment (lyuboxin) [reply test_addroom.py]: I will change it to 19487

**Review by zhangjiangqige — APPROVED** [review]

**Comment by xingweitian** [comment]
Please be careful when writing test case, since you are using the database which is in our production environment, do not insert garbage data into it.

**Comment by xingweitian** [comment]
Furthermore, do we need another database for web testing? @zhangjiangqige

**Comment by lyuboxin** [comment]
> Please be careful when writing test case, since you are using the database which is in our production environment, do not insert garbage data into it.

**Comment by zhangjiangqige** [comment]
> Furthermore, do we need another database for web testing? @zhangjiangqige

That'll be hard for frontend. I think a better way is to write a script which clears the database and inserts proper data (reset the database by running it).

**Review by xingweitian — APPROVED** [review]

**Pull request merged**